packages/mds-policy-linter/index.ts

```diff
@@ -0,0 +1,128 @@
+#!/usr/local/bin/node
+/**
+ * Copyright 2022 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PolicyDomainCreateModel, validatePolicyDomainModel } from '@mds-core/mds-policy-service'
```

Comment:
Looks like we need to get your ESLint/Prettier working correctly. This check failure should have been automatically fixed.